backend/app/core/orchestration/queue_manager.py

```
@@ -0,0 +1,132 @@
+                item = await self._get_next_item()
+
+                if item:
+                    await self._process_item(item, worker_name)
+                else:
+                    # No items available, wait a bit
+                    await asyncio.sleep(0.1)
+
+            except asyncio.CancelledError:
+                logger.info(f"Worker {worker_name} cancelled")
+                break
+            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)